contracts/mixnet/src/queries.rs

```diff
@@ -786,6 +840,248 @@ mod tests {
         )
     }
 
+    #[cfg(test)]
+    mod querying_for_mixnode_reverse_delegations_paged {
+        use super::*;
+        use crate::storage::mix_reverse_delegations;
+
+        fn query_mixnode_reverse_delegation(
+            deps: Deps,
+            mix_identity: IdentityKey,
+            address: Addr,
+        ) -> Result<(), ContractError> {
+            mix_reverse_delegations_read(deps.storage, &address)
+                .may_load(mix_identity.as_bytes())?
+                .ok_or(ContractError::NoMixnodeDelegationFound {
```

Comment:
Is `NoMixnodeDelegationFound` the correct error type for what you're doing here?